### ISSUE TITLE
Terrible VoiceOver accessibility on the Mac

### ISSUE THREAD

**Issue by 76534** [pinokiocomputer/pinokio#165]
I downloaded this app thinking it would be a cool way to use AI on my Mac without using for example python but VoiceOver is does not work with this app at all when it does it let me download things but I don't know for example where to enter text it just says terminal input which is not very useful because I'm not sure if I'm supposed to type there or it's doing something else Can you please fix this?

**Comment by techshare101** [comment]
No good
